Setup/UpgradeData.php

```diff
@@ -206,35 +210,19 @@ public function updateSchemaVersion800(ModuleDataSetupInterface $setup)
      *
      * @param ModuleDataSetupInterface $setup
      */
-    public function updateSchemaVersion821(ModuleDataSetupInterface $setup)
+    public function updateSchemaVersion820(ModuleDataSetupInterface $setup)
```

Comment:
for those who have already upgraded to 8.2.0, will they not miss this script when they if they upgrade to 8.2.1?

Reply:
Good point. This was a mistake on my part in the previous release. Ultimately it should not cause any failures since we only did it for precaution. Updated in https://github.com/Adyen/adyen-magento2/pull/1447/commits/8803da18cca78588d36efd8efaea93e237a74e70. Hence once 8.2.1 is out, everything should be inline once users upgrade to this version.